src/server/pfs/server/driver.go

```diff
@@ -118,7 +119,8 @@ func newDriver(env *serviceenv.ServiceEnv, txnEnv *txnenv.TransactionEnv, etcdPr
 	if err != nil {
 		return nil, err
 	}
-	chunkStorage := chunk.NewStorage(objClient, chunk.NewPostgresStore(db), tracker, chunkStorageOpts...)
+	memCache := kv.NewMemCache(10)
```

Comment:
We should probably go ahead and setup the configuration for this.

Reply:
`StorageMemoryCacheSize` denominated in chunks, same as `StorageDiskCacheSize`